maitake/src/wait/dataqueue.rs

```diff
@@ -166,6 +166,7 @@ mod tests;
 /// [ilist]: cordyceps::List
 /// [intrusive]: https://fuchsia.dev/fuchsia-src/development/languages/c-cpp/fbl_containers_guide/introduction
 /// [2]: https://www.1024cores.net/home/lock-free-algorithms/queues/intrusive-mpsc-node-based-queue
+#[derive(Debug)]
```

Comment:
nit: this debug impl still requires `K` and `V` to be `Debug`, which it maybe shouldn't?

Reply:
Does it? Waiter has a custom debug impls, which I think covers it? But I'll make sure before marking this ready.